### DISCUSSION TITLE
Setting requested flat heat temperature

### DISCUSSION THREAD

**Discussion by Mesztica** [Egyras/HeishaMon#581] [Q&A]
I'm trying to set the heating temperature but I fail. According to the documentation there is MQTT command to do it:

**SET5 | SetZ1HeatRequestTemperature | Set Z1 heat shift or direct heat temperature | -5 to 5 or 20 to max**

It works fine as shift in the range -5...5, but when I try to set fixed 30 degree it always goes back to -5...5 range. 
Are there any other settings enabling flat temp value? Im using Curve at the moment, so maybe I need to disable it first? But how?

**Comment by geduxas** [comment]
it's depends to heating mode, if you in compensation curve, it will always be -5..5 range.

**Reply by Mesztica** [reply]
Great, so it looks I need to change heating mode to "direct", I will do it in the device since i dont see MQTT command for that.
Thank you!

**Comment by krzbor** [comment]
The use of a heating curve offers more flexibility than a constant temperature. Two points on the heating curve can be adjusted through HeishaMon. This allows you to work with the curve, and if the target temperatures for both points on the curve are the same, you achieve a constant heating temperature.
